src/utilstrencodings.cpp

```diff
@@ -544,3 +548,13 @@ bool ParseFixedPoint(const std::string &val, int decimals, int64_t *amount_out)
     return true;
 }
 
+#ifdef WIN32
+std::wstring Utf8ToWide(const std::string& utf8_string)
+{
+    size_t size = MultiByteToWideChar(CP_UTF8, 0, &*utf8_string.begin(), utf8_string.size(), nullptr, 0);
+    std::wstring wide_string(size, 0);
+    size = MultiByteToWideChar(CP_UTF8, 0, &*utf8_string.begin(), utf8_string.size(), &*wide_string.begin(), wide_string.size());
+    assert(size == wide_string.size());
```

Comment:
Is this assert necessary? MultiByteToWideChar won't change the size of the string.

Reply:
They won't change, so that's why I use assert instead of a bunch of error handling.

Reply:
I didn't spot the size= part before the asset. Sorry :)